examples/bank/bank.go

```diff
@@ -159,22 +160,28 @@ func (bank *Bank) continuousMoneyTransfer(cash int64) {
 	}
 }
 
-// Initialize all the bank accounts with cash.
-// When multiple instances of the bank app are running,
-// only one gets to initialize the bank.
-func (bank *Bank) initBankAccounts(cash int64) {
+// Initialize all the bank accounts with cash.  When multiple
+// instances of the bank app are running, only one gets to initialize
+// the bank.
+func (bank *Bank) initBankAccounts(cash int64) int64 {
+	var initCash int64
 	if err := bank.db.Tx(func(tx *client.Tx) error {
-		// Check if the accounts have been initialized by another instance
-		if scan, err := tx.Scan(bank.makeAccountID(0), bank.makeAccountID(bank.numAccounts), int64(bank.numAccounts)); err != nil {
+		// Check if the accounts have been initialized by another instance.
+		scan, err := tx.Scan(bank.makeAccountID(0), bank.makeAccountID(bank.numAccounts), int64(bank.numAccounts))
+		if err != nil {
 			return err
-		} else if len(scan.Rows) == bank.numAccounts {
-			log.Warning("accounts have already been initialized")
-			return nil
-		} else if len(scan.Rows) > 0 {
-			// TODO(vivek): recover from this error
-			return fmt.Errorf("%d of %d accounts of the database are initialized", len(scan.Rows), bank.numAccounts)
 		}
-		// Let's initialize all the accounts
+		// Determine existing accounts.
+		existAcct := &Account{}
+		accts := map[string]struct{}{}
+		for i := range scan.Rows {
+			accts[string(scan.Rows[i].Key)] = struct{}{}
+			if err := existAcct.decode(scan.Rows[i].ValueBytes()); err != nil {
+				log.Fatal("error decoding existing account %s: %s", scan.Rows[i].Key, err)
+			}
+			initCash += existAcct.Balance
+		}
```

Comment:
Feels a bit overkill to allow to banking example instances to run simultaneously on the same set of accounts. Is there a particular need for this? If you want additional load on a cluster I'd arrange for the different instances to use unique prefixes.